math/vecops/inc/ROOT/RVec.hxx

```diff
@@ -1242,6 +1242,52 @@ RVec<T> Where(const RVec<int>& c, T v1, T v2)
    return r;
 }
 
+/// Return the angle difference (delta phi) of two scalars.
+/// The function computes the closest angle from v1 to v2 with sign and is
+/// therefore in the range [-M_PI, M_PI].
+/// The computation is done per default in radians (c == M_PI) but can be switched
+/// to degrees (c == 180).
+template <typename T>
+T DeltaPhi(T v1, T v2, const T c = M_PI)
+{
+   static_assert(std::is_floating_point<T>::value,
+                 "DeltaPhi must be called with floating point values.");
+   auto r = std::fmod(v2 - v1, 2.0 * c);
+   if (r < -c) {
+      r += 2.0 * c;
+   }
+   else if (r > c) {
+      r -= 2.0 * c;
+   }
+   return r;
+}
+
+/// Return the angle difference (delta phi) in radians of two vectors.
+template <typename T>
+RVec<T> DeltaPhi(const RVec<T>& v1, const RVec<T>& v2, const T c = M_PI)
+{
+   using size_type = typename RVec<T>::size_type;
+   const size_type size = v1.size();
+   auto r = RVec<T>(size);
+   for (size_type i = 0; i < size; i++) {
+      r[i] = DeltaPhi(v1[i], v2[i], c);
+   }
+   return r;
+}
+
+/// Return the angle difference (delta phi) in radians of a vector and a scalar.
+template <typename T>
+RVec<T> DeltaPhi(const RVec<T>& v1, T v2, const T c = M_PI)
```

Comment:
Do we need also a simple forwarder with the signature `RVec<T> DeltaPhi(T, const RVec<T>& , const T)`

Reply:
I put it in!